R/fragments-label.R

```diff
@@ -72,9 +72,11 @@
 ##'
 ##' ## Annotate the spectum with the fragment labels
 ##' plotSpectra(sp, labels = labelFragments, labelPos = 3)
-labelFragments <- function(x, tolerance = 0, ppm = 20, ...) {
+labelFragments <- function(x, tolerance = 0, ppm = 20,
+                           what = c("ion", "mz"), ...) {
```

Comment:
`what` is missing from the roxygen header. It should also be illustrated in an example.